anvil/core/src/eth/mod.rs

```diff
@@ -573,14 +589,14 @@ mod tests {
 
     #[test]
     fn test_custom_auto_mine() {
-        let s = r#"{"method": "evm_setAutomine", "params": [false]}"#;
+        let s = r#"{"method": "anvil_setAutomine", "params": [false]}"#;
```

Comment:
instead of replacing the method name, just duplicate these lines and then use new alias so that we have full coverage, same for all the other variants